services/libs/tinybird/pipes/pull_request_analysis_copy_pipe.pipe

```diff
@@ -13,42 +13,53 @@ SQL >
         memberId,
         organizationId
     FROM activityRelations_deduplicated_cleaned_ds
-    WHERE type = 'pull_request-opened'
+    WHERE type = 'pull_request-opened' OR type = 'merge_request-opened' OR type = 'changeset-created'
 
 NODE pull_request_first_assigned
 SQL >
     SELECT sourceParentId, MIN(timestamp) AS assignedAt
     FROM activityRelations_deduplicated_cleaned_ds
-    WHERE type = 'pull_request-assigned'
+    WHERE type = 'pull_request-assigned' OR type = 'merge_request-assigned'
     GROUP BY sourceParentId
     order by min(timestamp) desc
 
 NODE pull_request_first_review_requested
 SQL >
     SELECT sourceParentId, MIN(timestamp) AS reviewRequestedAt
     FROM activityRelations_deduplicated_cleaned_ds
-    WHERE type = 'pull_request-review-requested'
+    WHERE type = 'pull_request-review-requested' OR type = 'merge_request-review-requested'
```

Comment:
Follow up question on this one. Since for gerrit we won't have these 2 metrics, how do we handle these cases? Would it mean that we would never show Gerrit metrics for the "Merge lead time" widget?